scripts/mkcloud

```diff
@@ -798,6 +798,11 @@ function onhost_reset_admin
     safely bootstrapcrowbar
 }
 
+function save_cloudsource
```

Comment:
You may be right, but since the separation of `mkcloud` and `qa_crowbarsetup.sh` I'm not sure the `cloudsource` variable is rightly exported. This is why I set this next to the variable declaration.

It is true though that the original place were `/etc/cloudsource` is written is in `qa_crowbarsetup.sh`. Maybe can be done there. @jdsn what's your opinion?